database/engine/README.md

```diff
@@ -6,17 +6,88 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/database/engine/
 
 # Database engine
 
-The Database Engine works like a traditional database. It dedicates a certain amount of RAM to data caching and
+The Database Engine works like a traditional database. It dedicates a certain amount of RAM for data caching and
 indexing, while the rest of the data resides compressed on disk. Unlike other [database modes](/database/README.md), the
 amount of historical metrics stored is based on the amount of disk space you allocate and the effective compression
```

Comment:
Too much detail for an opening paragraph. Consider removing it completely. The amount of metrics stored is more complex than this anyway.